app/views/ContactGEEKView.coffee

```diff
@@ -0,0 +1,39 @@
+require('app/styles/contact-geek.sass')
+RootView = require 'views/core/RootView'
+template = require 'templates/contact-geek-view'
+utils = require 'core/utils'
+storage = require 'core/storage'
+
+
+module.exports = class ContactGEEKView extends RootView
+  id: 'contact-geek-view'
+  template: template
+
+  events:
+    'click .one': 'onClickOne'
+    'click .two': 'onClickTwo'
+
+  initialize: (options) ->
+    super(options)
+    @history = @getRedirect()
+    console.log @history
+    if @history
+      setTimeout @goRedirect, 5000
+
+  goRedirect: (value) ->
+    @redirect = utils.getQueryVariable 'redirect'
```

Comment:
@redirect is for global variable, 
for local variable, just use `redirect`, or `redirectURL`